### PR TITLE
April cleanup

### PR DESCRIPTION
**Pull request by russgold**
Add unit tests and start refactoring the Domain presence logic

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,6 +9,7 @@ ENV PATH=$PATH:/operator
 
 COPY src/scripts/* /operator/
 COPY operator/target/weblogic-kubernetes-operator-0.2.jar /operator/weblogic-kubernetes-operator.jar
+COPY target/weblogic-deploy.zip /operator/weblogic-deploy.zip
```
Comment (rjeberhard) [inline Dockerfile]: This should stay in feature/wdt

File: operator/src/main/java/oracle/kubernetes/operator/KubernetesConstants.java [hunk 2]
```diff
@@ -26,5 +26,6 @@
   public static final String CONTAINER_NAME = "weblogic-server";
   
   public static final String DOMAIN_CONFIG_MAP_NAME = "weblogic-domain-cm";
+  public static final String DOMAIN_HOME_CONFIG_MAP_NAME = "weblogic-domain-home-cm";
```
Comment (rjeberhard) [inline operator/src/main/java/oracle/kubernetes/operator/KubernetesConstants.java]: Needs to stay on feature/wdt

File: operator/src/main/java/oracle/kubernetes/operator/ProcessingConstants.java [hunk 3]
```diff
@@ -31,6 +31,7 @@
   public static final String EXPLICIT_RESTART_CLUSTERS = "explicitRestartClusters";
   
   public static final String SCRIPT_CONFIG_MAP = "scriptConfigMap";
+  public static final String DOMAIN_HOME_CONFIG_MAP = "domainHomeConfigMap";
```
Comment (rjeberhard) [inline operator/src/main/java/oracle/kubernetes/operator/ProcessingConstants.java]: Stays in feature/wdt

File: pom.xml [hunk 4]
```diff
@@ -49,6 +49,23 @@
 
     <build>
         <plugins>
+            <plugin>
```
Comment (rjeberhard) [inline pom.xml]: Leave on feature/wdt

**Comment by russgold** [comment]
wdt changes have been removed

**Pull request closed without merging**